File: all/abc295/abc295_a/abc295_a.java
import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();
    String[] s = new String[n];
    for(int i = 0; i < n; i++) {
      s[i] = stdIn.next();
    }

    String[] ptn = {"and", "not", "that", "the", "you"};
    for(int i = 0; i < n; i++) {
      for(int j = 0; j < ptn.length; j++) {
        if(s[i].equals(ptn[j])) {
          System.out.println("Yes");
          System.exit(0);
        }
      }
    }

    System.out.println("No");
  }
}

File: all/abc295/abc295_a/abc295_a.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int n;
  cin >> n;

  vector<string> w(n);
  for(auto &z : w) cin >> z;

  bool ans = false;
  for(auto &z : w) {
    if(z == "and" || z == "not" || z == "that" || z == "the" || z == "you") ans = true;
  }

  cout << (ans ? "Yes" : "No");
}